css/order.css

```diff
@@ -258,9 +250,166 @@ svg {
 }
 /** Hide Panels by Default **/
 
-#blockonomics_checkout:not(.no-js) .bnomics-order-container .bnomics-order-expired-wrapper,
 #blockonomics_checkout:not(.no-js) .bnomics-order-container .bnomics-order-panel,
 #blockonomics_checkout:not(.no-js) .bnomics-order-container .bnomics-copy-amount-text,
 #blockonomics_checkout:not(.no-js) .bnomics-order-container .bnomics-copy-address-text {
   display: none;
 }
+
+/* ----- Blockonomics Font Icons ----*/
+
+@font-face {
+  font-family: 'blockonomics-icons';
+  src: url('../fonts/blockonomics-icons.woff?6m1shn') format('woff');
+  font-weight: normal;
+  font-style: normal;
+  font-display: block;
+}
+
+[class^="blockonomics-icon-"], [class*=" blockonomics-icon-"] {
+  /* use !important to prevent issues with browser extensions that change fonts */
+  font-family: 'blockonomics-icons' !important;
+  speak: never;
+  font-style: normal;
+  font-weight: normal;
+  font-variant: normal;
+  text-transform: none;
+
+  /* Better Font Rendering =========== */
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+
+  margin-left: 2px;
+  margin-right: 2px;
+  font-size: 1.5em;
+}
+
+[class^="blockonomics-icon-"].spin, [class*=" blockonomics-icon-"].spin {
+  animation-name: rotate;
+  animation-duration: 2s;
+  animation-iteration-count: infinite;
+  animation-timing-function: linear;
+}
+
+@keyframes rotate {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+.blockonomics-icon-cart::before {
+  content: "\e900";
+}
+
+.blockonomics-icon-qr::before {
+  content: "\e901";
+}
+
+.blockonomics-icon-copy::before {
+  content: "\e902";
+}
+
+.blockonomics-icon-refresh::before {
```

Comment:
Yes, updated this, I had to convert the icon downloaded from figma from Strokes to Fills because icomoon wasn't supporting it directly, it's updated now. 

For future reference, if you see this: 
![image](https://user-images.githubusercontent.com/22245433/209075691-108c8ab0-8b37-4105-992b-dca75530a912.png)

Use this website https://iconly.io/tools/svg-convert-stroke-to-fill to convert the icon and then use it in IcoMoon, the documentation in icomoon asks you to do in Illustrator and other professional tools, but if it's a simple icon use this!